portfolio/src/main/webapp/script.js

```diff
@@ -78,3 +78,26 @@ fetch('/data').then(response => response.json()).then((myData) => {
     const arrayContainer = document.getElementById('array-container');
     arrayContainer.innerText = myData; });
 }
+
+/*
+ * Fetches the comments array and adds each element to the history 
+ * unordered list on the homepage by calling addCommentToDom
+ */
+ function getComments(){
+     fetch('/data').then(response => response.json()).then((comments) => {
+        const historyEl = document.getElementById('history');
+        console.log(comments + " and " + comments.length);
+        addCommentToDom(historyEl, comments);
+    });
+ }
+
+ /*
+  * Creates elements out of array to add to page in an unordered list
+  */
+function addCommentToDom(element, commentArray){
```

Comment:
nitpick again: addCommentToDom would add a single comment to the Dom, this function iterates through a list of comments. I would do the iteration outside of this function (for comment in array, addCommentToDom()) OR you could rename the function to more accurately describe what it is doing, but I would lean towards the first option :)